tests/regression_tests/rotation/inputs_true.dat

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Doesn't look like this file is used? The rotation test is just a normal (non Python API) test.

Reply:
Thanks! looks like this was leftover from my update to the new `develop` history after getting rid of that `plot_settings.pkl` file.